README.md

```diff
@@ -43,13 +43,7 @@ Follow these instructions to run ODK Central Frontend in development. For deploy
 
 First, run ODK Central Backend.
 
-Next, build ODK Central Frontend files for development by running `npm run dev`. The files will be outputted to `dist/`. As you update the source code, the files will be automatically rebuilt.
-
-Finally, run NGINX by changing the working directory to the root directory of the repository, then typing the following:
-
-```bash
-nginx -c "$PWD/nginx.conf" -p "$PWD"
-```
+Next, start nginx and build ODK Central Frontend files for development by running `npm run dev`. The files will be outputted to `dist/`. As you update the source code, the files will be automatically rebuilt.
```

Comment:
With the thought that not all frontend devs will know nginx, I would maybe explain that part a little more. But not overly highlight it either, since part of the beauty of Foreman is that the dev doesn't really need to understand nginx. How about:

> Next, build ODK Central Frontend files for development by running `npm run dev`. The files will be outputted to `dist/`. As you update the source code, the files will be automatically rebuilt. `npm run dev` will also start NGINX, which will serve the files.

Reply:
LGTM :+1: